xep-0320.xml

```diff
@@ -292,7 +303,7 @@ a=setup:role
                 <xs:enumeration value='active'/>
                 <xs:enumeration value='passive'/>
                 <xs:enumeration value='actpass'/>
-                <xs:enumeration value='holdconn'/>
+                <xs:enumeration value='holdconn'/><!-- not used -->
```

Comment:
It's probably more schema-ish to put this commented text into an xs:annotation element.